nemo/collections/nlp/models/language_modeling/megatron_t5_model.py

```diff
@@ -104,6 +104,9 @@ def build_train_valid_test_datasets(self):
             eval_iters * global_batch_size,
             test_iters * global_batch_size,
         ]
+        # Make sure the user specifies dataset type as either 't5' or 't5_prefix_lm' only.
+        if self._cfg.data.get('dataset_type', None) is not None:
+            assert self._cfg.data.get('dataset_type') in ['t5', 't5_prefix_lm']
```

Comment:
Following guidelines, perhaps replace with raising an error? (not sure about this one)